llama_stack/distribution/build_container.sh

```diff
@@ -43,7 +43,7 @@ RED='\033[0;31m'
 NC='\033[0m' # No Color
 
 CONTAINER_BINARY=${CONTAINER_BINARY:-docker}
-CONTAINER_OPTS=${CONTAINER_OPTS:-}
+CONTAINER_OPTS=${CONTAINER_OPTS:---progress=plain}
```

Comment:
We should also check `CONTAINER_BINARY` and skip if it's podman?

Reply:
It’s not a crucial flag; I’d simply remove it and let users add it back via CONTAINER_OPTS if they really need it. So that we avoid the extra overhead of checking `CONTAINER_BINARY`.

Also my podman version 5.2.0 does not complain about this flag... Maybe just give it an update?

Reply:
I'm on latest Podman (5.4.1)